yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -866,23 +910,11 @@ object Client extends Logging {
    * Get the list of namenodes the user may access.
    */
   private[yarn] def getNameNodesToAccess(sparkConf: SparkConf): Set[Path] = {
-    sparkConf.get("spark.yarn.access.namenodes", "")
-      .split(",")
-      .map(_.trim())
-      .filter(!_.isEmpty)
-      .map(new Path(_))
-      .toSet
+    SparkHadoopUtil.get.asInstanceOf[YarnSparkHadoopUtil].getNameNodesToAccess(sparkConf)
```

Comment:
Wonder if `YarnSparkHadoopUtil` should have its own `get`, to avoid callers having to do this cast.